components/PixTutorial.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <section class="tuto">
```

Comment:
```suggestion
  <section>
```
Je n'ai pas l'impression que cette classe soit utile hormis nommage pour BEM.

Reply:
C'est suffisant non ? ;)